platform/web/package.json

```diff
@@ -1,24 +1,25 @@
 {
-  "name": "godot",
-  "private": true,
-  "version": "1.0.0",
-  "description": "Development and linting setup for Godot's Web platform code",
-  "author": "Godot Engine contributors",
-  "license": "MIT",
-  "scripts": {
-    "docs": "jsdoc --template js/jsdoc2rst/ js/engine/engine.js js/engine/config.js js/engine/features.js --destination ''",
-    "lint": "cd ../.. && eslint --no-config-lookup --config ./platform/web/eslint.config.cjs ./platform/web/js ./modules ./misc/dist/html",
-    "format": "npm run lint -- --fix"
-  },
-  "devDependencies": {
-    "@eslint/js": "^9.12.0",
-    "@html-eslint/eslint-plugin": "^0.27.0",
-    "@html-eslint/parser": "^0.27.0",
-    "@stylistic/eslint-plugin": "^2.9.0",
-    "eslint": "^9.12.0",
-    "eslint-plugin-html": "^8.1.1",
-    "espree": "^10.0.1",
-    "globals": "^15.9.0",
-    "jsdoc": "^4.0.3"
-  }
+	"name": "godot",
+	"private": true,
+	"version": "1.0.0",
```

Comment:
Is this a node/npm update that causes the indentation change from 2 spaces to a tab? Or is that a personal IDE config?

We should probably make sure we pick a standard and stick to it to avoid having needless diffs. I'm fine changing to tabs if this is the recommended way, as long as we don't change it back again.

Reply:
This was part of @adamscott's changes, so I'm not sure the motivation, but for now, I've switched it back to spaces - it makes the diff much easier to review